src/library_wasi.js

```diff
@@ -33,6 +33,7 @@ var WasiLibrary = {
         'PWD': '/',
         'HOME': '/home/web_user',
         'LANG': lang,
+        'TZ': emscriptenGetTimeZone(),
```

Comment:
We should probably put this up in the `#if DETERMINISTIC` block above and pick a fixed timezone for the deterministic case.